doc/api/child_process.md

```diff
@@ -299,6 +299,9 @@ output on this fd is expected to be line delimited JSON objects.
 *Note: Unlike the fork(2) POSIX system call, `child_process.fork()` does
 not clone the current process.*
 
+*Note*: Support for non-Array values of the `.stdio` option was added in
```

Comment:
@addaleax (because you did the original YAML markup, I think) @MylesBorins Will this work?

Our 'ADDED' metadata is not granular enough. It tracks API additions, mostly, but it doesn't account for backporting though 00ffa335641b20118fed6ae9b005154fa28e1ea7 gives me the impression it was intended to.

Since it doesn't track changes in APIs, will this note work?  Will the FIXME get processed?